src/app/drive/components/DriveExplorer/DriveExplorerItem/DriveExplorerListItem/DriveExplorerListItem.tsx

```diff
@@ -136,12 +148,12 @@ const DriveExplorerListItem = ({ item }: DriveExplorerItemProps): JSX.Element =>
       }
 
       {/* DATE */}
-      <div className="hidden w-3/12 items-center overflow-ellipsis whitespace-nowrap lg:flex">
+      <div className="w-3/12 min-w-date items-center overflow-ellipsis whitespace-nowrap lg:flex">
         {dateService.format(item.updatedAt, 'DD MMMM YYYY. HH:mm')}
       </div>
 
       {/* SIZE */}
-      <div className="flex w-1/12 items-center overflow-ellipsis whitespace-nowrap">
+      <div className="w-1/12 min-w-breadcrumb items-center overflow-ellipsis whitespace-nowrap">
```

Comment:
Header and column should be aligned. The slight misalignment happens because the w- 1/12, w-3/12, etc, always use flex and absolute values, avoid using %. Video attached for more context.

https://github.com/internxt/drive-web/assets/34165099/d0131408-3f84-4144-ad08-c203bff9362c